source/linux/How_to_Guides/Target/How_to_emmc_boot.rst

```diff
@@ -0,0 +1,288 @@
+      |      tiboot3.bin (1 MB)          |         |                         |
+      +----------------------------------+0x400    |                         |
+      |       tispl.bin (2 MB)           |         |                         |
+      +----------------------------------+0x1400   |                         |
+      |       u-boot.img (4 MB)          |         |                         |
+      +----------------------------------+0x3400   |                         |
+      |      environment (128 KB)        |         |                         |
+      +----------------------------------+0x3500   |                         |
+      |   backup environment (128 KB)    |         |                         |
+      +----------------------------------+0x3600   +-------------------------+
+                   Boot0 (8 MB)                              UDA
+
+.. ifconfig:: CONFIG_part_variant not in ('AM64X', 'J7200', 'J721S2', 'AM62X', 'AM62PX', 'AM62AX', 'AM62LX')
+
+   .. code-block:: text
+
+
+      +----------------------------------+0x0      +-------------------------+0x0
+      |      tiboot3.bin (512 KB)        |         |                         |
+      +----------------------------------+0x400    |                         |
+      |       tispl.bin (2 MB)           |         |                         |
+      +----------------------------------+0x1400   |                         |
+      |       u-boot.img (4 MB)          |         |                         |
+      +----------------------------------+0x3400   |                         |
+      |      environment (128 KB)        |         |                         |
+      +----------------------------------+0x3500   |                         |
+      |   backup environment (128 KB)    |         |                         |
+      +----------------------------------+0x3600   |                         |
+      |          sysfw (1 MB)            |         |                         |
+      +----------------------------------+0x3E00   +-------------------------+
+                   Boot0 (8 MB)                              UDA
+
+Flash bootloader binaries to eMMC
+=================================
+
+To boot with *eMMC boot* the eMMC needs to be prepared before hand. The recommended process
+is to flash an SD card with TI SDK image, copy the bootlader binaries that will be flashed to
+eMMC in the SD card "boot" partition in emmc/ folder, boot the board with *MMCSD boot* from SD
+(FS mode), and proceed to flash the eMMC from either u-boot or linux.
+
+**Flash from u-boot**
+
+Stop at u-boot prompt and flash eMMC using :command:`fatload` and :command:`mmc write` commands
+to load binaries from SD card and flash them to eMMC Boot0.
+
+In this example, eMMC device is :command:`dev 0`, to find which device is eMMC, refer to
+:ref:`this <uboot-listing-mmc-devices>`. section. To flash to eMMC Boot1 instead, replace
+:command:`mmc dev 0 1` with :command:`mmc dev 0 2`.
```

Comment:
The command role is good for highlighting key words and drawing attention to details about tooling, but actual command examples and args should be
```
``wrapped``
```
so that they appear as highlight-able code snippets.